<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>de-extinct</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300;400&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="assets/css/globals.css">
    <link href="dist/output.css" rel="stylesheet">
</head>

<body class=" bg-slate-50 pb-24 w-full">
    <header class="w-full fixed px-6 sm:px-8 bg-slate-50 bg-opacity-95 z-50 h-16 top-0">
        <nav class="w-full flex py-3 items-center">
            <div class="grow">
                <a href="/bio309-final/" class="hover:no-underline">
                    <h3>de-extinct</h3>
                </a>
            </div>
            <div>
                <ul class="list-none space-x-2 nav-links">
                    <li class="inline-block"><a href="./about.html">About</a></li>
                    <li class="inline-block"><a href="./techniques.html">Techniques</a></li>
                    <li class="inline-block"><a href="./ethics.html">Ethics</a></li>
                </ul>
            </div>
        </nav>
    </header>
    <div class="content-holder non-hero">
        <section class="space-y-3 my-3">
            <h1 class="mb-3">technology</h1>
            <h2>sequencing extinct genomes</h2>
            <p>
                DNA degrades over time, in part due to a process called cytosine deamination. This process occurs when a
                cytosine undergoes hydrolysis, and is converted to a uracil. If left unchecked, this creates G to A and
                C to T mutations in the resulting reads [1]. As such, cytosine deamination is a major
                inhibitor to obtaining high-quality genomes for extinct species, and processes need to be in place to
                address its effects.
            </p>
            <p>
                Paleogenomics comprises a set of techniques for sequencing ancient genomes through the careful
                extraction of preserved DNA samples. In contrast to modern genome sequencing from live samples,
                paleogenomic research must take abundant precautions in order to avoid damaging samples. The risk of
                losing a sample is far greater, as there is only a finite—and usually miniscule—amount of DNA that is
                preserved well enough to be studied. Samples usually come from fossils or remains that have been
                well-preserved by their environment. For example, one group of researchers located preserved mammoth
                bone samples and found a tooth with around 80% mammoth DNA content [2].To then mitigate potential
                effects of cytosine deamination, the researchers used an enzyme to remove uracils in the DNA. They
                obtained DNA sequence reads, giving the researchers a complete genome to further analyze. In theory,
                with more complete genomes, a mammoth reference genome could be created—significantly contributing to
                mammoth de-extinction efforts.
            </p>
            <p>
                For more recently extinct species, well-preserved specimens may exist, enabling higher-quality
                sequencing without additional paleogenomic steps. One example is the Tasmanian Tiger: a recent study
                seeking to assemble the Tasmanian Tiger's genome used samples from a 108 year old ethanol-preserved
                specimen [3]. Researchers used genomic analysis programs to quantify cytosine deamination in the
                Tasmanian tiger genome. However, they found no evidence of deamination, likely due to the relatively
                recent death of the sample organism, especially in comparison to the mammoth samples in the previous
                study.
            </p>
        </section>
        <section class="space-y-3 my-3">
            <h2>applications of CRISPR</h2>
            <p>
                CRISPR-Cas9 is an endonuclease system from bacteria that defends against bacteriophages. However, recent
                research has revealed that the CRISPR system can be used by researchers to artificially modify DNA in
                live cells with high accuracy [4]. The implications of this discovery for biological
                research are immense; for de-extinction work, this means that reintroducing lost alleles to living
                organisms is a viable way to increase biodiversity. CRISPR would also be a valuable tool in the
                potential reconstruction of extinct genomes. One company called Colossal is currently working to use
                CRISPR to stitch together DNA sequences from extinct organisms to construct a full genome (COLOSSAL
                CITATION HERE).
                They then plan to create artificial embryos of these species to rebirth extinct lineages.
            </p>
        </section>
        <section class="space-y-3 my-3">
            <h2>references</h2>
            <ol class="space-y-3">
                <li class="reference">[1] Dabney J., M. Meyer, and S. Paabo, 2013 Ancient DNA Damage. Cold Spring Harbor
                    Perspectives in
                    Biology 5: a012567-a012567. <a href="https://doi.org/10.1101/cshperspect.a012567"
                        target="_blank">https://doi.org/10.1101/cshperspect.a012567</a></li>
                <li class="reference">[2] Palkopoulou E., S. Mallick, P. Skoglund, J. Enk, N. Rohland, et al., 2015
                    Complete Genomes
                    Reveal Signatures of Demographic and Genetic Declines in the Woolly Mammoth. Current Biology 25:
                    1395-1400. <a href="https://doi.org/10.1016/j.cub.2015.04.007"
                        target="_blank">https://doi.org/10.1016/j.cub.2015.04.007</a>
                </li>
                <li class="reference">[3] Feigin C. Y., A. H. Newton, L. Doronina, J. Schmitz, C. A. Hipsley, et al.,
                    2017 Genome of the Tasmanian tiger provides insights into the evolution and demography of an extinct
                    marsupial carnivore. Nat Ecol Evol 2: 182-192. <a href="https://doi.org/10.1038/s41559-017-0417-y"
                        target="_blank">https://doi.org/10.1038/s41559-017-0417-y</a></li>
                <li class="reference">[4] Jinek M., K. Chylinski, I. Fonfara, M. Hauer, J. A. Doudna, et al., 2012 A
                    Programmable Dual-RNA-Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science 337: 816-821.
                    <a href="https://doi.org/10.1126/science.1225829">https://doi.org/10.1126/science.1225829</a>
                </li>
            </ol>
        </section>

    </div>

</body>

</html>